test/lib.app.js

```diff
@@ -40,14 +40,14 @@ describe('App method', () => {
         assert.equal(app.androidVersion, '2.3');
         assert.equal(app.androidVersionText, '2.3 and up');
 
-        assert.equal(app.priceText, 'Free');
+        assert.equal(app.priceText, '');
         assert.equal(app.price, 0);
         assert(app.free === true);
         assert.equal(app.offersIAP, false);
         // assert(app.preregister === false);
 
         assert.equal(app.developer, 'DxCo Games');
-        assert.equal(app.developerId, 'DxCo+Games');
+        assert.equal(app.developerId, '8699533450056637394');
```

Comment:
you changed this for the tests to pass, but it's actually a bug. Apparently the previous mapping doesn't work anymore. Please leave this test failing so we address it separately.

Reply:
Thanks for the review. ,I remember seeing the ID; But shall cross check again and update PR in couple of days.